File: Kap07/C/Listing36/SubStrExample.c
#include <stdio.h>

int main() {
  char stringVar[] = "hallo welt";
  char subStringVar[6];
  for (int i = 0; i < 5; i++) {
    subStringVar[i] = stringVar[i];
  }
  subStringVar[5] = 0;
  printf("Wert: %s", subStringVar);
  return 0;
};

File: Kap07/Java/Listing37/SubStrExample.java
public class SubStrExample {
    public static void main(String[] args) {
        String stringVar = "hallo welt";
        char[] stringVarArray = stringVar.toCharArray();
        char[] subStringArray = new char[5];
        for (int i = 0; i < 5; i++) {
        subStringArray[i] = stringVarArray[i];
        }
        String subStringVar = new String(subStringArray);
        System.out.println(subStringVar);
    }
}
    